Dispatcher/UI/Search/SearchScreenViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  SearchScreenViewController.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 02/05/2023.
+//
+
+import UIKit
+
+class SearchScreenViewController: UIViewController {
+
+    @IBOutlet weak var searchTableView: UITableView!
+    @IBOutlet weak var clearButton: UIButton!
+    @IBOutlet weak var babkButton: UIButton!
+    @IBOutlet weak var searchBar: UISearchBar!
+    
+    let searchScreenViewModel = SearchViewModel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        customSearchBar()
+       
+        searchTableView.dataSource = self
+        searchTableView.delegate = self
+        searchTableView.register(UINib(nibName: AppConstants.latestSearchCellNibName, bundle: nil), forCellReuseIdentifier: AppConstants.latestSearchesCellIdentifier)
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        searchScreenViewModel.fetchLatestSearchs()
+        searchTableView.reloadData()
+    }
+    
+    private func customSearchBar() {
+        let searchTextField = searchBar.searchTextField
+        searchBar.setImage(UIImage(named: "exit"), for: .clear, state: .normal)
+        
+        searchTextField.leftView = nil
+        searchTextField.backgroundColor = .white
+
+    }
+    
+    
+    @IBAction func backButtonPressed(_ sender: UIButton) {
+        self.dismiss(animated: true)
+    }
+    
+    @IBAction func clearButtonPressed(_ sender: UIButton) {
```

Comment:
in the context of the design of the screen it's not clear whether it's the x button of the searchbar or the square clear button of the saved searches.
So a better name will be "clearAllRecentSearchesBtnPressed" or something like that